client/app/components/account_config_input.coffee

```diff
@@ -0,0 +1,97 @@
+{div, label, input} = React.DOM
+{ErrorLine} = require './basic_components'
+classer = React.addons.classSet
+
+RouterMixin = require '../mixins/router_mixin'
+
+
+# Input used in the account configuration/creation form. An account input can
+# display an error message below it.
+module.exports = AccountInput = React.createClass
+    displayName: 'AccountInput'
+
+    mixins: [
+        RouterMixin
+        React.addons.LinkedStateMixin # two-way data binding
+    ]
+
+
+    getInitialState: ->
+        return @props
+
+
+    componentWillReceiveProps: (props) ->
+        @setState props
```

Comment:
Please change that part later if you want. I don't know the application well enough to change that properly.